src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -127,7 +165,7 @@ class ZWaveJSConfigDashboard extends LitElement {
                               ></ha-svg-icon>
                             `}
                       </div>
-                      ${this._status !== "connecting"
+                      ${this._status !== "disconnected"
```

Comment:
Can we add a stricter type to `_status `?

Reply:
good idea